4/docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: "3.8"
+services:
+  primarysilo:
+    build:  
+      context: .
+      dockerfile: ./ops/SiloHost/Dockerfile
+    environment:
+    - GATEWAYPORT=3000
+    - SILOPORT=2000
```

Comment:
This is definitely nitpicking, but let's reserve those nice, round numbers for single-silo scenarios. In multi-silo clusters, let's indicate the multitude of silos with numbers. Something like this:

* first silo:
  * gateway: 3001
  * siloport: 2001
* second silo:
  * gateway: 3002
  * siloport: 2002

etc.